### PR TITLE
Prepare 0.5.0 release

### PR DESCRIPTION
**Pull request by ackermannW**
Signed-off-by: Mirza Hodzic <mirza.hodzic@secomind.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.5.0] - 2023-04-06
```
Comment (harlem88) [inline CHANGELOG.md:7]: Change to today
```suggestion
## [0.5.0] - 2023-04-07
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+### Added
+- Initial Astarte Device C# SDK release
```
Comment (harlem88) [inline CHANGELOG.md:9]: Add a dot at the end of the line
```suggestion
- Initial Astarte Device C# SDK release.
```

**Review by harlem88 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ackermannW) [reply CHANGELOG.md]: Suggested changes were applied.

File: CHANGELOG.md (hunk 2 above)
Comment (ackermannW) [reply CHANGELOG.md]: Suggested changes were applied.

**Review by harlem88 — APPROVED** [review]

**Pull request merged**